notebooks/ch17/classification_multiclass.md

```diff
@@ -0,0 +1,18 @@
+
+## Multiclass Classification
+
+Should this be a separate section or just an addendum to an existing section?
```

Comment:
I think this makes sense as a (short) new section since the previous sections are full of information already.